content/sdk/80.troubleshooting.md

```diff
@@ -17,50 +17,31 @@ correct endpoint for the L2 network is: [ZKsync Sepolia Explorer](%%zk_testnet_b
+deprecation warnings during installation, follow these steps to troubleshoot and resolve the problems.
 
-If you encounter issues using Bun during the installation, refer to the troubleshooting steps in the table below.
+#### Step 1: Install the Missing Module
+You must install the required package if an error indicates a missing
+module (e.g., `@nomicfoundation/hardhat-toolbox`). Run the following command in your project directory:
 
-##### 1. Network timeout
+```sh
+npm install @nomicfoundation/hardhat-toolbox
+```
 
-- **Possible cause**: Slow or unstable internet connection
-- **Solution**: Increase the timeout duration by using: `curl -fsSL --connect-timeout 60 https://bun.sh/install | bash`
+#### Step 2: Address Deprecation Warnings
+During installation, you might see deprecation warnings. These warnings indicate that some packages are outdated but
+do not necessarily prevent your project from running. You can address them by updating the respective packages. However,
+these warnings can be ignored if they do not cause functional issues.
 
-##### 2. Manual download required
+#### Step 3: Run Hardhat
+After installing the required packages, you can verify the installation by running Hardhat:
 
-- **Possible cause**: Automatic download fails
-- **Solution**: Manually download the Bun binary from
-the [Bun releases page on GitHub](https://github.com/oven-sh/bun/releases/latest). Extract the binary and place it in a
-directory that is in your system's PATH.
+```sh
+npx hardhat
+```
 
-##### 3. Permissions issue
+#### Step 4: Compile the Project
+To ensure everything is set up correctly, try compiling your project:
 
-- **Possible cause**: Bun binary is not executable
-- **Solution**: Ensure the Bun binary is executable by running: `chmod +x ~/.bun/bin/bun`
-
-##### 4. PATH not updated
-
-- **Possible cause**: `~/.bun/bin` not in PATH
-- **Solution**: Verify and manually add it to your PATH by editing your shell configuration
-file (`~/.zshrc` for zsh): `echo 'export PATH="$HOME/.bun/bin:$PATH"' >> ~/.zshrc && source ~/.zshrc`
-
-##### 5. TTY initialization failed
-
-- **Possible cause**: Terminal handling issues or TTY not supported
-- **Solution**: Run without TTY by setting the environment variable: `NO_TTY=1 bun install` or disable
-telemetry: `export NUXT_TELEMETRY_DISABLED=1`
-
-##### 6. Telemetry prompt issue
-
-- **Possible cause**: Nuxt telemetry prompt causing issues
-- **Solution**: Set an environment variable to disable the telemetry prompt: `export NUXT_TELEMETRY_DISABLED=1`
-
-##### 7. Conflicting aliases
-
-- **Possible cause**: Conflicts with existing aliases or functions
-- **Solution**: Ensure there are no conflicting aliases or functions named `bun` by running: `type bun`
-
-##### 8. Reinitialize shell
-
-- **Possible cause**: Shell environment not refreshed
-- **Solution**: Completely reinitialize your shell environment by starting a new shell session: `exec /bin/zsh`
+```sh
+npx hardhat compile
```

Comment:
These seem like very vague steps to just getting a project running correctly. They don't have any particular error they're addressing. Let's remove these changes.